framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -56,6 +57,26 @@
     /** The empty range. */
     public static final Range NOTHING = new Range();
 
+    /**
+     * Convenience construtor for often invoked pattern for Long collections
```

Comment:
Please use correct grammar:  sentences end with a period.
Please say *what* the pattern is.  This documentation doesn't do that.
Please don't make value judgments like "often invoked" without evidence.  Just say what it is.

Reply:
Hi Michael.
Your help is most appreciated.
I will fix the syntax on a newer commit.
I am still getting the hang of the ticketing system here.
I tried documenting the "pattern" in the issue: 
https://github.com/typetools/checker-framework/issues/2905
The desire is to no longer have that in the client code using Range().

Documenting in javadoc what one wants to remove/go away/retire from the code seems odd.

The refactoring here was a "pull up" from the client code using Range() into convenience constructors in Range().

The examples of the repeated code being absorbed are found in the pull request.

I hope this helps.